social_media_pinterest/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Diagram Software S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': "Pinterest Social Media Icon Extension",
+    'summary': """Pinterest Extension for the social media icons from the
+    odoo core""",
+    'author': "Diagram Software, S.L."
+    "Odoo Community Association (OCA)",
+    'website': "http://www.diagram.es",
+    'license': 'AGPL-3',
+    'category': 'Social Media',
+    'version': '8.0.1.0.0',
+
+    'depends': [
+        'base',
+        'website',
+        'website_blog'
```

Comment:
I don't think it's wise to directly depend on website_blog. There are possibility that company want this but do not want the blog. You can create additional module to add this social media to blog.